source/clear-linux/guides/maintenance/Telemetry-Enable-Disable.rst

```diff
@@ -0,0 +1,111 @@
+.. _telemetry-enable-disable:
+
+Enable and Disable Telemetry in Clear Linux
+###########################################
+
+|CLOSIA| includes a telemetry solution as part of the OS that records events
+of interest and reports them back to the development team via the telemetrics
+daemon, :command:`telemd`.  This functionality is maintained in the
+``telemetrics`` software bundle.
+
+.. Note::
+	The telemetrics functionality adheres to `Intel’s privacy policies`_
+	regarding the collection and use of Personally Identifiable Information
+	(PII) and is Open Source. Specifically, no intentionally identifiable

```

Comment:
Low case for open source

Reply:
done.  Might want to check the clearlinux.org/features/telemetry doc for this as well.